corehq/motech/utils.py

```diff
@@ -10,15 +11,15 @@
 
 AES_BLOCK_SIZE = 16
 AES_KEY_MAX_LEN = 32  # AES key must be either 16, 24, or 32 bytes long
-PAD_CHAR = ' '
+PAD_CHAR = b' '
```

Comment:
Same question here - are you sure the tests shouldn't be updated instead?

Reply:
I think in this particular case, we could add an assert that `string` is a bytestring, because `blocksize` doesn't set the number of characters; it sets the number of bytes.